crux-mir/report-coverage/src/main.rs

```diff
@@ -747,35 +784,74 @@ impl Filter {
 
 
 fn report_all(reporter: &mut Reporter, cov: &Coverage) {
-    for (span, bcov) in cov.iter_sorted() {
-        let is_boolean = bcov.is_boolean &&
-            bcov.possible.iter().all(|&x| x == 0 || x == 1) &&
-            !bcov.default_possible;
-        if is_boolean {
-            if !bcov.seen.contains(&0) {
-                reporter.warn(span, "branch condition never has value false");
+
+    let mut summary = HashMap::new();
+
+    for (fun, fn_cov) in cov.iter_sorted() {
+        
+        // hacky: skip core-library or crucible functions
+        if fun.starts_with("core/") || fun.starts_with ("crucible/") { continue }
```

Comment:
Would the right way to do this be to only consider functions that come from the crate being tested itself?